packages/spark-core/spark-core.js

```diff
@@ -29,24 +29,27 @@ import './utilities/polyfills/classListSVG';
 // Utilities
 import detectJS from './utilities/detectJS';
 
-// Init
-detectJS();
-requiredSelect();
-requiredTick();
-requiredTextInput();
-ssnInput();
-passwordInput();
-monetaryInput();
-phoneInput();
-dateInput();
-datePicker();
-modals();
-pagination();
-tabs();
-toggle();
-spinners();
-WideNavigation();
-masthead();
-alerts();
-textInput();
-selectInput();
+const SparkCore = ({ datePickerConfig = {} } = {}) => {
+  detectJS();
```

Comment:
If we are removing the Init fns then do we need to update the getting started developers page?

Reply:
yep, will do